tests/samplers_tests/tpe_tests/test_parzen_estimator.py

```diff
@@ -204,3 +208,120 @@ def test_log_pdf_parzen_estimator(multivariate: bool) -> None:
     # The likelihood of the previous observations is a positive value, and that of the points
     # sampled by the Parzen estimator is almost zero.
     assert np.all(log_pdf >= output_log_pdf)
+
+
+@pytest.mark.parametrize(
+    "mus, prior, magic_clip, endpoints",
+    itertools.product(
+        (np.asarray([]), np.asarray([0.4]), np.asarray([-0.4, 0.4])),  # mus
+        (True, False),  # prior
+        (True, False),  # magic_clip
+        (True, False),  # endpoints
+    ),
```

Comment:
Just a comment but you can also apply `pytest.mark.parametrize` multiple times to a test function. That'll have the same product effect and might be more ideomatic.